test/common/fixtures/plugins/coverage/public/plugin.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Plugin } from 'kibana/server';
+
+interface WindowWithCoverage extends Window {
+  __coverage__: any;
+  flushCoverageToLog: any;
+}
+
+export class CodeCoverageReportingPlugin implements Plugin {
+  constructor() {}
+
+  public start() {}
+
+  public setup() {
+    (window as WindowWithCoverage & typeof globalThis).flushCoverageToLog = function () {
```

Comment:
is that not supposed to be double-and `&&`?

Reply:
Single-and is a bitwise op.

Reply:
I don't think it is needed here, since I'm not doing logical operation like `window && typeof window === "object"`
Just search a bit and found it [here](https://github.com/microsoft/TypeScript/issues/33128#issuecomment-527572596)

Reply:
Yeah, in type signatures a single `&` between two types creates a union type of the two.